kie-internal/src/main/java/org/kie/internal/process/CorrelationAwareProcessRuntime.java

```diff
@@ -56,6 +56,20 @@ ProcessInstance startProcess(String processId, CorrelationKey correlationKey,
     ProcessInstance createProcessInstance(String processId, CorrelationKey correlationKey,
                                           Map<String, Object> parameters);
 
+    /**
+     * Starts a process with the values supplied from the current nodes
+     * 
+     * @param deploymentId deployment information for the process's kjar
+     * @param processId The process's identifier
+     * @param correlationKey correlation key to be assigned to process instance - must be unique 
+     * @param params process variables
+     * @return process instance identifier
+     * @throws RuntimeException in case of encountered errors
+     * @throws DeploymentNotFoundException in case deployment with given deployment id does not exist
+     * @throws DeploymentNotActiveException in case deployment with given deployment id is not active
```

Comment:
Please correct javadocs. They don't match with what the method signature.

Reply:
done